featurebyte/config.py

```diff
@@ -522,11 +522,13 @@ def get_client(self) -> APIClient:
         APIClient
             API client
         """
-        # pylint: disable=import-outside-toplevel,cyclic-import
-        from featurebyte.logging import reconfigure_loggers
+        if not is_server_mode():
+            # pylint: disable=import-outside-toplevel,cyclic-import
+            from featurebyte.logging import reconfigure_loggers
+
+            # configure logger
+            reconfigure_loggers(self)
```

Comment:
@kchua78 This operation is pretty expensive (takes about 0.02s) and we call `Configurations().get_client()` so many times in SDK codes. Maybe we should cache this to reduce the runtime.